core/src/main/java/ch/cyberduck/core/Local.java

```diff
@@ -370,20 +370,20 @@ public void copy(final Local copy, final CopyOptions options) throws AccessDenie
             if(log.isDebugEnabled()) {
                 log.debug(String.format("Copy to %s with options %s", copy, options));
             }
-            InputStream in = null;
-            OutputStream out = null;
+            FileChannel c1 = null;
```

Comment:
Could you rename the local variable back to `in` and `out` respectively to make it clear what the source and target of the copy operation is.

Reply:
fixed